app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/TooltipCaptureTest.java

```diff
@@ -0,0 +1,95 @@
+import org.junit.Test;
+import org.mozilla.tv.firefox.MainActivity;
+import org.mozilla.tv.firefox.R;
+import org.mozilla.tv.firefox.helpers.MainActivityTestRule;
+
+import tools.fastlane.screengrab.locale.LocaleTestRule;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.RootMatchers.isPlatformPopup;
+import static androidx.test.espresso.matcher.ViewMatchers.hasFocus;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static org.hamcrest.Matchers.allOf;
+
+public class TooltipCaptureTest extends ScreenshotTest {
+
+    @ClassRule
+    public static final LocaleTestRule localeTestRule = new LocaleTestRule();
+
+    @Rule
+    public ActivityTestRule<MainActivity> mActivityTestRule = new MainActivityTestRule();
+
+    @After
+    public void tearDown() {
+        mActivityTestRule.getActivity().finishAndRemoveTask();
+    }
+
+    @Test
+    public void showToolTips() throws InterruptedException {
+
+        onView(allOf(withId(R.id.navUrlInput), hasFocus())).check(matches(isDisplayed()));
+        onView(withId(R.id.tileContainer)).check(matches(isDisplayed()));
+
+        // open two sites to enable back and front button
+        onView(withId(R.id.tileContainer)).perform(RecyclerViewActions.actionOnItemAtPosition(1, click()));
+        device.wait(Until.findObject(By.res(Integer.toString(R.id.progressAnimation))), 2000);
+        device.pressMenu();
+        onView(withId(R.id.tileContainer)).perform(RecyclerViewActions.actionOnItemAtPosition(2, click()));
+        device.wait(Until.findObject(By.res(Integer.toString(R.id.progressAnimation))), 2000);
+
+        device.pressMenu();
+        device.pressDPadUp();
+        onView(withId(R.id.tooltip)).inRoot(isPlatformPopup()).check(matches(isDisplayed()));
+        takeScreenshotsAfterWait("tooltip-backbutton", 500);
+
+        onView(withId(R.id.navButtonBack)).perform(click());
+        device.wait(Until.findObject(By.res(Integer.toString(R.id.progressAnimation))), 2000);
+        device.pressMenu();
+        device.pressDPadUp();
+        device.pressDPadRight();  // In fastlane, back button is still enabled - this might be fastlane issue
+
+        // forward button
+        onView(withId(R.id.tooltip)).inRoot(isPlatformPopup()).check(matches(isDisplayed()));
+        takeScreenshotsAfterWait("tooltip-forwardbutton", 500);
+        device.pressDPadRight();
+        onView(withId(R.id.tooltip)).inRoot(isPlatformPopup()).check(matches(isDisplayed()));
+        takeScreenshotsAfterWait("tooltip-reload", 500);
+        device.pressDPadRight();
+        onView(withId(R.id.tooltip)).inRoot(isPlatformPopup()).check(matches(isDisplayed()));
+        takeScreenshotsAfterWait("tooltip-pintohs", 500);
```

Comment:
nit: is this a typo for filename?

Reply:
ah actually that's a shorthand for "pinto**H**ome**S**creen"